<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-06-16T20:37:31 IST" name="Suite" finished-at="2023-06-16T20:37:45 IST" duration-ms="14043">
    <groups>
    </groups>
    <test started-at="2023-06-16T20:37:31 IST" name="ChromeTest" finished-at="2023-06-16T20:37:45 IST" duration-ms="14043">
      <class name="executeClasses.ExecuteClientsPage">
        <test-method is-config="true" signature="createReport(org.testng.ITestContext)[pri:0, instance:executeClasses.ExecuteClientsPage@5adb0db3]" started-at="2023-06-16T20:37:30 IST" name="createReport" finished-at="2023-06-16T20:37:31 IST" duration-ms="41" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@374ccb9]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createReport -->
        <test-method is-config="true" signature="beforeMethod(java.lang.String)[pri:0, instance:executeClasses.ExecuteClientsPage@5adb0db3]" started-at="2023-06-16T20:37:31 IST" name="beforeMethod" finished-at="2023-06-16T20:37:39 IST" duration-ms="8770" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyClientSearch()[pri:0, instance:executeClasses.ExecuteClientsPage@5adb0db3]" started-at="2023-06-16T20:37:39 IST" name="verifyClientSearch" finished-at="2023-06-16T20:37:43 IST" duration-ms="4042" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Appu] but found [Amala p]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Appu] but found [Amala p]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at executeClasses.ExecuteClientsPage.verifyClientSearch(ExecuteClientsPage.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyClientSearch -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:executeClasses.ExecuteClientsPage@5adb0db3]" started-at="2023-06-16T20:37:43 IST" name="afterMethod" finished-at="2023-06-16T20:37:45 IST" duration-ms="1152" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyClientSearch status=FAILURE method=ExecuteClientsPage.verifyClientSearch()[pri:0, instance:executeClasses.ExecuteClientsPage@5adb0db3] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- executeClasses.ExecuteClientsPage -->
    </test> <!-- ChromeTest -->
  </suite> <!-- Suite -->
</testng-results>
